Contents/mods/Hydrocraft/media/scripts/Mining.txt

```diff
@@ -823,6 +823,17 @@ recipe Dig Quarry
     	OnGiveXP:HCHunger_OnGiveXP,
     }
 
+recipe Dig Quarry
+    {
+        PickAxe,
```

Comment:
I think this would be ok, but would need to change the dig quarry to keep the auger, and the upgrade recipe from quarry to mine to consume an auger